backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/UnidentifiedSpecimenTypeException.java

```diff
@@ -16,7 +16,7 @@ public UnidentifiedSpecimenTypeException(String typeCode) {
     super(
         "Specimen with type code "
             + typeCode
-            + " weren't found. to make sure that type code exists");
+            + " wasn't found. to make sure that type code exists");
```

Comment:
Is the second part also a typo here? `Make sure that type code exists` instead of `to make sure that type code exists`?